tests/API/OVAL/unittests/test_state_check_existence.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+result=`mktemp`
+
+set -e
+set -o pipefail
+
+cat > /tmp/foo.txt <<EOF
```

Comment:
I don't have better idea than https://github.com/OpenSCAP/openscap/pull/154/files#r39490287  ...
I prefer something like that pseudo-bash:
cat \<oval file with relative paths\> | sed replace \<relative path =\> absolute path\> | \`mktemp\`

IMHO generating of whole oval file in bash isn't good solution.